src/main/java/seedu/address/model/group/GroupHashMap.java

```diff
@@ -46,7 +46,14 @@ public void add(Group toAdd) {
      * Replace given person name with new name in all groups.
      */
     public void replacePerson(Name personToReplace, Name newPersonName) {
-        internalMap.values().stream().forEach(g -> g.setPersonName(personToReplace, newPersonName));
+        internalMap.values().forEach(g -> g.setPersonName(personToReplace, newPersonName));
+    }
+
+    /**
+     * Deletes given person from all groups in {@code internalMap}.
```

Comment:
Document behaviour here also -> that the `personToDelete` is guaranteed to be removed from all groups.